development/run-version-bump.js

```diff
@@ -7,25 +7,28 @@ const changelogPath = path.join(__dirname, '..', 'CHANGELOG.md')
 const manifestPath = path.join(__dirname, '..', 'app', 'manifest.json')
 const manifest = require('../app/manifest.json')
 const versionBump = require('./version-bump')
-
 const bumpType = normalizeType(process.argv[2])
 
+start()
+.then(console.log)
+.catch(console.error)
 
-readFile(changelogPath)
-.then(async (changeBuffer) => {
-  const changelog = changeBuffer.toString()
+async function start() {
 
-  const newData = await versionBump(bumpType, changelog, manifest)
+  readFile(changelogPath)
```

Comment:
the point of moving this into an async fn is so you could use an `await` here : )

Reply:
oh lol I copy pasted good call.